modules/virt-deploying-operator-cli.adoc

```diff
@@ -0,0 +1,57 @@
+// Module included in the following assemblies:
+//
+// * virt/install/installing-virt-cli.adoc
+
+[id="virt-deploying-operator-cli_{context}"]
+= Deploying the {VirtProductName} Operator by using the CLI
+
+You can deploy the {VirtProductName} Operator by using the `oc` CLI.
+
+.Prerequisites
+
+* An active subscription to the {VirtProductName} catalog in the `openshift-cnv` namespace.
+
+.Procedure
+
+. Create a file named `hco_cr.yaml` by running the following command:
```

Comment:
Are the file names used in these modules fixed, or can I call them anything?

Reply:
file names does not matter. what's matter is only the content of the file.
it can be done even without creating a file, using:
```
cat <<EOF | oc apply -f -
    <file contents>
EOF
```